SConstruct

```diff
@@ -1751,6 +1752,7 @@ imageEnvAppends = {
 }
 
 imageEnv.Append( **imageEnvAppends )
+imageEnv["ENV"][libraryPathEnvVar] = imageEnv["LIBPATH"]
```

Comment:
@andrewkaufman Any thoughts on whether this will make things explode your end? We needed it as `libOpenImageIO.so` and `libfreetype.so` in the gaffer dependencies project needed it for their deps (png, etc..).

Reply:
It needs those at compile time or just at runtime? Is this change just to get passed the scons configure checks? If so, we've used this kind of workaround in the past:
ttps://github.com/ImageEngine/cortex/blob/master/SConstruct#L1939-L1949

Reply:
Ahh good trick, thanks. I've updated the FreeType one in https://github.com/ImageEngine/cortex/pull/988/commits/012edffda121f4550cf60d7cf0f01fbcef39c1e6, but I couldn't get that to work for the `imageEnv` version - ended up with `OpenImageIO_v1_8::ustring::empty_std_string` missing during tests. Any ideas there?

Reply:
Hmm, I wonder why we don't run into this issue at IE or on Travis... We were already running the IECoreImage tests on Travis, using the gaffer dependencies, right?